src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Config/SwaggerConfig.cs

```diff
@@ -83,7 +81,7 @@ public void Apply(OpenApiOperation operation, OperationFilterContext context)
             foreach (var parameter in operation.Parameters)
             {
                 var description = apiDescription.ParameterDescriptions
-                    .First(p => p.Name == parameter.Name);
+                    .First(p => p.Name.ToLower() == parameter.Name.ToLower());
```

Comment:
This is a little inefficient as we're performing some transformation instead of a straight comparison between the strings. 

A more efficient version of this would be:

```cs
.First(p => p.Name.Equals(parameter.Name, StringComparison.CurrentCultureIgnoreCase));
```